sigstore/root-signing
# setup-root for asraa

Signed-off-by: Asra Ali <asraa@google.com>



#### Summary

* This initialies the v4 root signing.

TO VERIFY:
```
./scripts/verify.sh 313
```

Please manually verify:
* There is a root and targets threshold of 3
* The **staged** `root.json` and `targets.json` are version 4
* Joshua's key ID `75e867ab10e121fdef32094af634707f43ddd79c6bab8ad6c5ab9f03f4ea8c90` is added to the root and targets keyholders
* Luke's is rotated out: `bdde902f5ec668179ff5ca0dabf7657109287d690bf97e230c21d65f99155c62`
* Top level **staged** targets includes the fulcio intermediate certificate pem. please verify this too by checking what we use in sigstore/sigstore @haydentherapper 

#### Release Note


#### Documentation
